object/service.proto

```diff
@@ -75,7 +124,17 @@ service ObjectService {
   // Statuses:
   // - **OK** (0, SECTION_SUCCESS):
   // data range of the object payload has been successfully read;
-  // - Common failures (SECTION_FAILURE_COMMON).
+  // - Common failures (SECTION_FAILURE_COMMON);
+  // - **CONTAINER_NOT_FOUND** (3072, SECTION_CONTAINER):
+  // object container not found;
+  // - **ACCESS_DENIED** (2048, SECTION_OBJECT):
+  // access to operation RANGE of the object is denied;
+  // - **OBJECT_NOT_FOUND** (2049, SECTION_OBJECT):
+  // object not found in container;
+  // - **TOKEN_EXPIRED** (4097, SECTION_SESSION):
+  // provided session token has expired;
+  // - **OBJECT_ALREADY_REMOVED** (2052, SECTION_OBJECT):
+  // the requested object has been marked as deleted.
```

Comment:
Sort them by section and/or id?

Reply:
By ID is useless IMO since there are "holes" in numbers. By section - agree, I'd be convenient.
https://github.com/nspcc-dev/neofs-api/issues/198